src/Components/test/E2ETest/Microsoft.AspNetCore.Components.E2ETests.csproj

```diff
@@ -7,12 +7,16 @@
     <TargetFramework>netcoreapp3.0</TargetFramework>
     <TestGroupName>Components.E2ETests</TestGroupName>
 
-    <SkipTests Condition="$(ContinuousIntegrationBuild) == 'true'" >true</SkipTests>
     <!-- https://github.com/aspnet/AspNetCore/issues/6857 -->
     <BuildHelixPayload>false</BuildHelixPayload>
+    
+    <!-- Run on platforms where we support Selenium -->
+    <SkipTests Condition="'$(SeleniumE2ETestsSupported)' != 'true'">true</SkipTests>
```

Comment:
What changed for these not to hang, are you just giving it a go?

Reply:
I don't believe there ever was a hang. I think what happened was the scripts were out of date causing each test to timeout. It takes 30s for each our tests to timeout and we have hundreds of E2E tests.

That's why I sent that other PR that makes them fail fast 😆